CONTRIBUTING.md

```diff
@@ -185,7 +185,14 @@ If you are updating tests and just want to run a single test to check it, you
 can use this syntax to run it exactly as the test harness would:
 
 ```text
-$ python tools/test.py -v --mode=release parallel/test-stream2-transform
+$ python tools/test.py --mode=release parallel/test-stream2-transform
```

Comment:
Can we add a `-J`? Given that it says "exactly as the test harness would" we should probably be exact:

```diff
-$ python tools/test.py --mode=release parallel/test-stream2-transform
+$ python tools/test.py -J --mode=release parallel/test-stream2-transform
```